packages/sirius-web/backend/sirius-web-domain/src/main/java/org/eclipse/sirius/web/domain/services/api/IMessageService.java

```diff
@@ -57,6 +57,54 @@ public interface IMessageService {
 
     String unauthorized();
 
+    String studioTemplateName();
```

Comment:
While it's not an issue for this PR, we will definitely have to split this interface soon. `sirius-web-domain` should not depend on concerns such as diagram, studio, etc.